### PR TITLE
about artist page

### PR DESCRIPTION
**Pull request by jethoo**
### What this PR does (required):
- This is the artist about page.
- This page will contain personal info about the artist.
- Route to '/artist' will load this page
- PS: need to send artistId to '/artist' as a state property through Link (react router)

### Screenshots / Videos (required):
Screen Shot 2021-07-27 at 13 34 50
Screen Shot 2021-07-27 at 13 34 57

### Other changes 
- Created controller and route to get personal info based on the artistId provided

### Any information needed to test this feature (required):
- Post what steps to take to be able to test this feature (eg. "sign up as a new user and upload a profile picture from the navbar")

### Any issues with the current functionality (optional):
- List any problems where you were not able to complete all tasks on ticket
- List any problems this PR may cause for the project or other active PRs
- Post a screenshot/video of the problem, along with detailed console outputs where applicable

### CODE REVIEW

File: server/models/PersonalInfo.js [hunk 1]
```diff
@@ -3,7 +3,8 @@ const { Schema, model } = require('mongoose');
 const PersonalInfoSchema = new Schema({
   userId: {
     type: Schema.Types.ObjectId,
-    ref: 'user',
+    //ref: 'user',
```
Comment (rajivtitus) [inline server/models/PersonalInfo.js]: Please remove

File: server/controllers/personalInfo.js [hunk 2]
```diff
@@ -26,3 +26,15 @@ exports.getPersonalInfo = asyncHandler(async (req, res) => {
     res.status(500).json(err);
   }
 });
+
+exports.getInfoById = asyncHandler(async (req, res) => {
+  try {
+    const info = await PersonalInfo.findOne({
+      userId: req.params.id
+    }).populate('userId')
```
Comment (rajivtitus) [inline server/controllers/personalInfo.js]: While we are populating the user document, won't that return the user's password as well?

**Review by rajivtitus — COMMENTED** [review]
Looks good for a start!

A couple of comments, please have a look. Thanks

File: client/src/components/AboutArtistTab/AboutArtistTab.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+interface Props {
+  info: PersonalInfo | undefined | null;
+}
+
+const AboutArtistTab = ({ info }: Props): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <Box textAlign="center">
+      {info !== null && info !== undefined ? (
```
Comment (rajivtitus) [inline client/src/components/AboutArtistTab/AboutArtistTab.tsx:18]: Eventually we may need to change this logic when we have reviews available to us

File: server/controllers/personalInfo.js (hunk 2 above)
Comment (jeewanbhusal-1) [reply server/controllers/personalInfo.js]: Yes, it did return. I am changing to populate only profilePic in the next commit :)

File: client/src/components/AboutArtistTab/AboutArtistTab.tsx (hunk 3 above)
Comment (jeewanbhusal-1) [reply client/src/components/AboutArtistTab/AboutArtistTab.tsx:18]: yes, we never discussed about how we are dealing with reviews. apparently dealt this way.

File: server/models/PersonalInfo.js (hunk 1 above)
Comment (jeewanbhusal-1) [reply server/models/PersonalInfo.js]: done :)

**Review by rajivtitus — APPROVED** [review]
LGTM! Thanks for making requested changes

**Comment by jeewanbhusal-1** [comment]
> LGTM! Thanks for making requested changes

Thank you.

**Pull request merged**